service/ansible/roles/atmo-fail2ban/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+ssh_port: '22'
+ignoreip: '127.0.0.1/8 128.196.38.0/25 128.196.172.128/25 150.135.113.224/27 150.135.93.128/25 150.135.78.0/24 10.130.5.128/25 10.130.4.128/25 10.130.64.0/24'
```

Comment:
Another example of stuff that is specific only to our installation, to be pulled out in a future release.